alkos/kernel/arch/x86_64/boot64.nasm

```diff
@@ -0,0 +1,48 @@
+          ; Stack
+          extern stack_bottom
+          extern stack_top
+
+          ; GDT64
+          extern GDT64.Data
+
+          ; Kernel Entry Point
+          extern kernel_main
+
+          global boot64
+          section .text
+          bits 64
+boot64:
+          mov ax, GDT64.Data
+          mov ds, ax
+          mov es, ax
+          mov fs, ax
+          mov gs, ax
+          mov ss, ax
+
+          call kernel_main
+          mov r10, rax
+
+          ; clear screen
+          mov edi, 0xb8000
+          mov ecx, 80*25
+          xor eax, eax
+          rep stosd
+
+          mov rdi, 0xb8000 ; VGA text buffer
+          mov rcx, 80 * 25 ; 80 columns, 25 rows
+          mov ah, 0x08 ;
```

Comment:
This will literally be deleted in the next step. The only reason it's here, is because I wanted to know if it the jump to 64 bit code actually works. This should be immediately deleted and moved to the actual kernel in cpp